packages/browser/src/integrations/breadcrumbs.ts

```diff
@@ -75,9 +74,7 @@ export class Breadcrumbs implements Integration {
     const breadcrumb = {
       category: 'console',
       data: {
-        extra: {
-          arguments: normalize(handlerData.args, 3),
-        },
+        arguments: handlerData.args,
```

Comment:
This is "kinda" breaking if someone is relying on such an odd data point, but if we won't change it, it'll have same issue as the one described for `ExtractErrorData`